Source/swiftlint/Helpers/LintOrAnalyzeArguments.swift

```diff
@@ -42,6 +42,8 @@ struct LintOrAnalyzeArguments: ParsableArguments {
     var baseline: String?
     @Option(help: "The path to save detected violations to as a new baseline.")
     var writeBaseline: String?
+    @Option(help: "The working directory to use when running SwiftLint.")
```

Comment:
Maybe this should elaborate a bit more on what "working directory" means. Or is this obvious?

Reply:
Well, "working directory" has it's own wikipedia page (https://en.wikipedia.org/wiki/Working_directory) which is more than we do right now (just checked and my submission is still in review).

```
% man pwd
PWD(1)                                                                         General Commands Manual                                                                         PWD(1)

NAME
     pwd – return working directory name
```

So for at least some users, I think it should be obvious